src/main/java/dev/slimevr/gui/VRServerGUI.java

```diff
@@ -239,7 +241,9 @@ public void mouseClicked(MouseEvent e) {
 					}
 				});
 				setSkatingReductionEnabled(
-					(boolean) server.config.getProperty("legTweaks.skatingCorrection")
+					server.humanPoseProcessor
+						.getSkeletonConfig()
+						.getToggle("Skating Correction")
```

Comment:
Use the enum instead of a hardcoded string
```suggestion
						.getToggle(SkeletonConfigToggle.SKATING_CORRECTION)
```